content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -185,4 +175,23 @@ Insurance | Do you have insurance? | 1 Yes
 Mobile_money_classification |   | 0 no mobile money and no other financial service (saving, borrowing, insurance)
 |       |               | 1 no mobile money, but at least one other financial service
 |       |               | 2 mobile money only
-|       |               | 3 mobile money and at least one other financial service
\ No newline at end of file
+|       |               | 3 mobile money and at least one other financial service
+
+
+## Instructions:
+
+1. Import the required libraries and load the datadatset [training](training.csv).
```

Comment:
Good work so far. There is a typo here Neerja. Please work on it.

Reply:
You can install a spellchecker in vscode. I use 
Code Spell Checker
v2.20.5
Street Side Software
streetsidesoftware.com

Reply:
Also, is step 1 useful? It seems very obvious... Do the learners get confused if we dont tell them to do this? Maybe they do.

And do the learners know that they are meant to hand in a notebook? Are there any requirements around directory structures or anything like that?

Reply:
Sure, I will do that.